README.md

```diff
@@ -56,12 +56,6 @@ Additionally, when pushing commits for a recipe that excludes Windows, put `[ski
 on Windows from even starting.
 
 
-### 4. **What does `numpy x.x` mean?**
```

Comment:
We should either point people to the docs and rename entry `4` or re-number everything. I confess I am more inclined to the latter to keep the README short and add a "bigger" link to the docs at the top.

Reply:
OK, went with the second option. Renumbered other FAQ elements, and added a link to the docs in the beginning. Though maybe it's not visible enough...

Reply:
Thanks! That is enough for this PR.
We never advertised the docs properly and it is about time we do it.